tests/scripts/all.sh

```diff
@@ -2610,6 +2610,20 @@ component_test_tls13_experimental_with_everest () {
     if_build_succeeded tests/ssl-opt.sh
 }
 
+component_test_tls13_experimental_with_rsa () {
```

Comment:
MBEDTLS_RSA_C and MBEDTLS_X509_RSASSA_PSS_SUPPORT are enabled in the default configuration thus I do not think we need this additional component. `component_test_tls13_experimental` does the job.